core/src/main/java/org/apache/accumulo/core/conf/SiteConfiguration.java

```diff
@@ -110,8 +118,15 @@ private static URL toURL(File f) {
     }
   }
 
-  public static URL getAccumuloSiteLocation() {
-    String configFile = System.getProperty("accumulo.configuration", "accumulo-site.xml");
+  public static URL getAccumuloPropsLocation() {
+
+    URL siteUrl = SiteConfiguration.class.getClassLoader().getResource("accumulo-site.xml");
+    if (siteUrl != null) {
+      throw new IllegalArgumentException("Found deprecated config file 'accumulo-site.xml' on "
+          + "classpath. Since 2.0.0, this file was replaced by 'accumulo.properties'.");
+    }
+
+    String configFile = System.getProperty("accumulo.configuration", "accumulo.properties");
```

Comment:
This seems to imply that the `--props` argument is redundant. It might be confusing if we have too many different ways to do this. Personally, I prefer the `-Daccumulo.configuration=...` method, although perhaps it could be called `-Daccumulo.properties=...` instead for naming consistency and clarity?